packages/valory/skills/liquidity_rebalancing_abci/test_tools/integration.py

```diff
@@ -22,8 +22,8 @@
 from pathlib import Path
 from typing import Any
 
-from autonomy.test_tools.fixture_helpers import HardHatAMMBaseTest
```

Comment:
This was a big (undetected) problem. Deploying these as a service would not have worked in the new deployment flow.